package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "UKVI-Complaints",
+  "version": "0.0.1",
+  "main": "app.js",
+  "description": "UKVI Complaints form",
+  "license": "GPLv2",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/UKHomeOffice/UKVI-Complaints"
+  },
+  "scripts": {
+    "start": "node .",
+    "dev": "npm-run-all --parallel watch:app watch:scss watch:js watch:translations",
+    "watch:app": "NODE_ENV=development nodemon .",
+    "watch:scss": "nodemon -e scss -x 'npm run sass'",
+    "watch:js": "nodemon --watch assets/js -x 'npm run browserify'",
+    "watch:translations": "nodemon --watch apps/complaints/translations/src -x 'npm run hof-transpile'",
+    "test": "NODE_ENV=test mocha",
+    "browserify": "browserify ./assets/js/index.js > ./public/js/bundle.js",
+    "test:ci": "npm run lint && npm run style && npm run test",
+    "lint": "eslint .",
+    "copy:images": "cp -r ./assets/images ./public/",
+    "style": "jscs **/*.js --config=./.jscsrc",
+    "sass": "node-sass ./assets/scss/app.scss ./public/css/app.css --include-path ./node_modules",
+    "create:public": "mkdir -p ./public/js ./public/css ./public/images",
+    "hof-transpile": "hof-transpiler ./apps/**/translations/src -w --shared ./apps/common/translations/src",
+    "postinstall": "npm run create:public; npm run sass; npm run browserify; npm run copy:images; npm run hof-transpile"
+  },
+  "author": "HomeOffice",
+  "dependencies": {
+    "body-parser": "^1.13.0",
+    "browserify": "^12.0.1",
+    "connect-redis-crypto": "^2.1.0",
+    "cookie-parser": "^1.3.5",
+    "express": "^4.12.4",
+    "express-partial-templates": "^0.1.0",
+    "express-session": "^1.11.3",
+    "hof": "^7.1.0",
+    "hogan-express-strict": "^0.5.4",
+    "hogan.js": "^3.0.2",
+    "lodash": "^4.13.1",
+    "moment-business": "^2.0.0",
```

Comment:
I'm not 100% sure what this is doing. It's referenced in app.js. `require('moment-business');` but is not assigned to a variable and isn't referenced anywhere else. Could it be removed or am I missing the point of what it actually does?